edx_proctoring/templates/proctoring/proctored-exam-status.underscore

```diff
@@ -0,0 +1,13 @@
+<div class="exam-timer">
+<%- gettext("You are taking") %>
+<a href="<%- interpolate(gettext('%(exam_url_path)s'), { exam_url_path: exam_url_path }, true)%>" >
+
+<%- interpolate(gettext(' %(exam_display_name)s '), { exam_display_name: exam_display_name }, true) %>
+</a>
+<%- gettext(" exam as a proctored exam. Good Luck!") %>
```

Comment:
Suggest striking the first occurrence of "exam" after inserting the exam name. Courses might name their exam "Final exam" and then we'd have "You are taking Final exam exam as a proctored exam." :-)